DEVGUIDE.md

```diff
@@ -124,7 +124,7 @@ Running any of the above will build the latest changes and run tests against the
 
 ## Using your custom compiler to build this repository
 
-By removing all the subfolders called `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
+By removing all the subfolders called `Bootstrap` under `artifacts` and running the `build` script again, the proto (= bootstrap) compiler will include your changes.
```

Comment:
```suggestion
By removing all the subfolders called `Bootstrap` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
```

Proto = compiler configuration in solution. Bootstrap = process.

Reply:
Yeah, correct. The one in /fsc is called Proto.
![image](https://github.com/dotnet/fsharp/assets/5451366/4085938a-8286-4f35-88ae-a2a504320a81)

I put both names there then. As for proto vs bootstrap - would be useful to explain and uniform it. E.g. in the build script, `proto` is an alias for `bootstrap`.

But I just wanted to put the right folder name at least here :)

Reply:
`bootstrap` is a process, as in "we need to bootstrap Proto compiler", Proto is name of the configuration in solution.